R/regressionloglinearbayesian.R

```diff
@@ -41,11 +41,11 @@ RegressionLogLinearBayesian <- function(jaspResults, dataset = NULL, options, ..
     if(options$counts != "")
       counts <- options$counts
     if(length(options$modelTerms) > 0)
-      factors <- options$modelTerms
-    return(.readDataSetToEnd(columns.as.factor = factors, 
+      factors <- options$factors
```

Comment:
This change is unrelated to the fixes. I just noticed that the using the `modelTerms` we unnecessarily duplicate columns in the data set if interactions are specified in the model.